lcoe_calculator/tech_processors.py

```diff
@@ -88,26 +103,107 @@ class UtilityPvPlusBatteryProc(TechProcessor):
         ('PV System Cost ($/kW)', 'df_pv_cost'),
         ('Battery Storage  Cost ($/kW)', 'df_batt_cost'),
         ('Construction Finance Factor', 'df_cff'),
+        ('PV-only Capacity Factor (%)','df_pvcf')
     ]
 
+    def __init__(
+        self,
+        data_workbook_fname: str,
+        case: str = MARKET_FIN_CASE,
+        crp: CrpChoiceType = 30,
+        tcc: str = "PV PTC and Battery ITC",
+        extractor: type[AbstractExtractor] = PVBatteryExtractor
```

Comment:
type should be capitalized, but see other comments

Reply:
New idea: change this to `extractor: Type[PVBatteryExtractor] = PVBatteryExtractor`. It's painfully obvious it needs to be the special extractor or a derivative, and no changes needed anywhere else.